src/main/java/io/kamax/matrix/room/RoomTag.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * matrix-java-sdk - Matrix Client SDK for Java
+ * Copyright (C) 2018 Arne Augenstein
+ *
+ * https://www.kamax.io/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.kamax.matrix.room;
+
+public class RoomTag {
+
+    private String namespace;
+    private String name;
+    private Double order;
+    private String roomId;
+
+    public RoomTag(String namespace, String name, Double order, String roomId) {
+        this.namespace = namespace;
+        this.name = name;
+        this.order = order;
+        this.roomId = roomId;
+    }
+
+    public String getNamespace() {
```

Comment:
We might want to use `Optional<String>` as return type on this, but the spec is quite obscure on how the namespace should *really* work... To discuss.

Reply:
Seems to me, the namespace is mandatory. [The relevant part of the spec](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#id75) mentions tags without namespaces. But these tags seem only to be supported for historical reasons.

Reply:
What would this method return in case of a tag without a namespace in your interpretation of the spec?

Reply:
An empty string. Do you think that's wrong?

Reply:
Lesser of the evils, so let's go for an empty string, yes.